jwst/resample/resample_step.py

```diff
@@ -79,6 +79,7 @@ def process(self, input):
         for reftype in self.reference_file_types:
             ref_filename = self.get_reference_file(input_models[0], reftype)
 
+        ref_filename='N/A'
```

Comment:
Might want to add a comment here that this line will need to get removed when good drizpars ref files get delivered to CRDS and we want to start using them.